src/accelerate/accelerator.py

```diff
@@ -421,10 +427,30 @@ def __init__(
                 )
 
         self.device_placement = device_placement
-        self.split_batches = split_batches
-        self.dispatch_batches = dispatch_batches
-        self.even_batches = even_batches
-        self.use_seedable_sampler = use_seedable_sampler
+        if dataloader_config is None:
+            self.dataloader_config = DataLoaderConfiguration()
+        # Deal with deprecated args
+        deprecated_dl_args = {}
+        if dispatch_batches is not None:
+            deprecated_dl_args["dispatch_batches"] = dispatch_batches
+            self.dataloader_config.dispatch_batches = dispatch_batches
+        if split_batches is not True:
```

Comment:
shouldnt we check if this value is not `None` (similar to lie 434) ?

Reply:
No, only for ones whose value changes from the default. While dispatch_batches is None by default, split_batches (and more) are not

Reply:
`split_batches` is `False` by default though.

Reply:
Changed with recommendations to be a sentinel value